### ISSUE TITLE
a11y addon tests don't run on story load or change

### ISSUE THREAD

**Issue by mattrothenberg** [storybooks/storybook#5754]
**Describe the bug**
When using the `a11y` addon, I expect the tests for a particular story to run automatically on initial page load, and then on any subsequent story change.

I am using the global decorator, not individual decorators per components.

Currently, I have to manually press **Rerun test** in order for the tests to run at all.

**To Reproduce**
1. Spin up Storybook
2. Visit http://localhost:9001
3. Observe that there are no violations OR passes
4. Press **Rerun Test**
5. See that there are now violations/passes.

**Expected behavior**
I expect the tests to be run automatically when a story loads.

**System:**
 - OS: 10.14.3
 - Device: 2018 MBP
 - Browser: Chrome 72.0.3626.109
 - Framework: React
 - Addons: `a11y`
 - Version: 4.2.0-alpha.10

**Additional context**
I'm not using the latest version in the `4` series, as per this issue: https://github.com/storybooks/storybook/issues/5078

**Comment by mattrothenberg** [comment]
FWIW, this is now working beautifully after updating to latest release candidate ❤️
